### DISCUSSION TITLE
It isn't downloading everything off a newtumbl blog and I'm not sure why.

### DISCUSSION THREAD

**Discussion by Zelarian** [TumblThreeApp/TumblThree#433] [Q&A (Question & Answer)]
It says, Download images "117 of 421". I have everything ticked for downloading. The download process doesn't freeze or anything, when I add it to the queue it scans and completes normally like other blogs but it doesn't download the things that are missing. Other blogs are downloading fine, the posts are not missing in the blog itself. I don't have "Save url list" ticked. I don't have "Check for file existence globally across all loaded blogs" ticked. When I click "Dump crawler data" it shows the crawler files for the missing posts but doesn't download the posts themselves. There is nothing coming up in the log for this. The LastId is not 0 it is 195706351. I have no idea what is wrong. Thank you.

**Comment by Zelarian** [comment]
I think I figured it out! Windows file names cannot contain \ / : * ? " < > | these symbols. All the posts I checked that were missing had " or ? symbols in the titles, and I was trying to save the posts using the titles as the file names.

Could the program be written to automatically exclude those specific symbols when writing file names? I'm using %p%x for the file naming.

Edit: Possibly anyway... I found more posts that didn't have those symbols and still didn't download.

Edit 2: I think maybe the caption is too long? So that the filenames wouldn't be able to be saved? Even though the %p should be shorted to fit... lol I really don't know.

Edit 3: It is definitely something to do with the naming. I downloaded the blog twice, once with %p%x and once with %f. With %f it downloads everything, with %p%x it skips a lot of posts.

**Comment by thomas694** [comment]
Sure, but invalid characters for file names are replaced, e.g. `"Title with # : < > ?".jpg` is converted to `Title with # - &lt; &gt; -.jpg`. And titles that are too long are automatically shortened. If there's no title available the files are saved as `_1.jpg`, `_2.jpg`, ...

For your tests also make sure that the setting "check for file existence globally" isn't activated, else duplicates are not downloaded again.

If it still doesn't work for you, you can send us the blog URL together with "433" in the feedback dialog and we'll check if there's a problem with this blog.

**Comment by thomas694** [comment]
We checked it, there's indeed a problem on some systems if the title is too long. It will be fixed in the next version.

**Reply by Zelarian** [reply]
Thank you ^_^

**Reply by thomas694** [reply]
version 2.10.1 is released

**Reply by Zelarian** [reply]
Thank you for your work
